opentracing-api/pom.xml

```diff
@@ -29,8 +29,8 @@
 
     <properties>
         <main.basedir>${project.basedir}/..</main.basedir>
-        <main.java.version>1.6</main.java.version>
-        <main.signature.artifact>java16</main.signature.artifact>
+        <main.java.version>1.7</main.java.version>
```

Comment:
Yeah, makes sense. The 1.7 feature doesn't really change the fundamentals, it just allows for a friendlier syntax... so we could try to create both `opentracing-api` and `opentracing-api-1.6` artifacts, with the former built on top of the latter "in some way." I'm waving my hands about the details, though... I can imagine it creating lots of compatibility/versioning headaches downstream.